helm/postfix/values.yaml

```diff
@@ -45,13 +45,24 @@ service:
 # SMTP server details
 # Used by postfix to connect to SMTP server
 smtp:
+  
+  existingSecretName: postfix
   # Example with mailgun
   relayHost: "[smtp.mailgun.org]:587"
   relayMyhostname: my.host.local
   relayUsername: relayuser
   relayPassword: relaypassword
   relayNetworks: '10.0.0.0/8,127.0.0.0/8,172.17.0.0/16,192.0.0.0/8'
 
+# Use Externaly manages secrets
+useExistingSecret: false
+existingSecretName: postfix
+
+# e.g kuberentes-external-secrets
+useExternalSecrets: false
```

Comment:
Can we put this under a dedicated block like the following
```
externalSecret:
  enabled: false
  secretsBackendType: secretsManager
  secretsKey: infra/postfix
```

Reply:
done